Client/Frontend/Home/CustomizeHome/CustomizeHomepageSectionCell.swift

```diff
@@ -10,7 +10,7 @@ class CustomizeHomepageSectionCell: UICollectionViewCell, ReusableCell {
 
     private struct UX {
         static let buttonFontSize: CGFloat = 15
-        static let buttonTrailingSpace: CGFloat = 12
+        static let buttonTrailingSpace: CGFloat = 18
```

Comment:
The `buttonTrailingSpace` should not be changed. Could you please undo this?

Reply:
Okay, I will roll this change back

Reply:
Done ✅ Could you please review it again 🙏

Reply:
LGTM thanks for the contribution, I will just wait for @thatswinnie approval to merge

Reply:
@AhmadKhattab thanks for the quick change. Unfortunately our build system detected a SwiftLint error on line 52 (the argument is not aligned correctly). Could you please fix this? 🙏

Reply:
Sure, It's fixed now